setup.py

```diff
@@ -11,4 +11,13 @@
 with open("requirements.txt") as handle:
     REQUIRES = list(handle.read().splitlines())
 
-setup(install_requires=REQUIRES)
+setup(
+    name="weblate_schemas",
+    entry_points={
+        "fedora.messages": [
+            "base.message=weblate_schemas.messages:BaseMessage",
+            "weblate.message=weblate_schemas.messages:WeblateV1Message",
```

Comment:
I recommend suffixing this key with `.v1`, as it is the version 1 of the schema.